rest/route/github.go

```diff
@@ -302,7 +290,7 @@ func (gh *githubHookApi) handleCheckRunRerequested(ctx context.Context, event *g
 			"repo":    repo,
 			"message": "check run doesn't carry task",
 		})
-		return gimlet.NewJSONInternalErrorResponse(errors.New("check run doesn't carry task"))
+		return errors.New("check run doesn't carry task")
```

Comment:
Sorry, I know you didn't add this now, but should we maybe phrase this something like "check run from github does not have task associated with it"?

Reply:
added